services/movie_session.py

```diff
@@ -0,0 +1,59 @@
+from django.db.models import QuerySet
+from django.forms import DateTimeField
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: DateTimeField,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(
+        session_date: str = None,
+
+) -> QuerySet:
+
+    query_set = MovieSession.objects.all()
+    if session_date:
+        query_set.filter(show_time=session_date)
```

Comment:
The `filter` method is not applied to the `query_set`. You need to assign the result back to `query_set` to apply the filter.